WebApp/handler/ComponentHistoryHandler.py

```diff
@@ -0,0 +1,132 @@
+from WebAppDIRAC.Lib.WebHandler import WebHandler, WErr, WOK, asyncGen
+from DIRAC import gConfig, S_OK, S_ERROR, gLogger
+from DIRAC.Core.Utilities import Time
+from DIRAC.FrameworkSystem.Client.ComponentMonitoringClient import ComponentMonitoringClient
+import json
+import datetime
+
+class ComponentHistoryHandler( WebHandler ):
+
+  AUTH_PROPS = 'authenticated'
+
+  @asyncGen
+  def web_getInstallationData( self ):
+
+    req = self.__request()
+
+    client = ComponentMonitoringClient()
+    result = client.getInstallations( req[ 'installation' ], req[ 'component' ], req[ 'host' ], True )
```

Comment:
This call will block Tornado.
result = yield self.threadTask(client.getInstallations, req['installation'], req['component'], req['host'], True)